targets/hasura/migrations/default/1698219011954_update_document_highlight/up.sql

```diff
@@ -0,0 +1,4 @@
+update documents
+set "document" = jsonb_set("document", '{highlight}','{"title": "A noter : à partir du 01/01/2024 les informations ci-dessous ne seront plus applicables","content": "En effet, jusqu’au 31/12/2023, ce sont les conventions locales de la métallurgie ainsi que la convention spécifique aux ingénieurs et cadres de la métallurgie qui s’appliquent. Toutes ces conventions vont être regroupées pour former la convention collective nationale de la métallurgie (IDCC 3248) à partir du 01/01/2024."}', true)
+where "source" = 'contributions'
```

Comment:
```suggestion
where "source" = 'conventions_collectives'
```

Il me semble que c'est pas sur les contributions qu'on rajoute l'hightlight

De plus il faut vérifier qu'il y a pas déjà un highlight dans la convention collective